.github/workflows/ci.yaml

```
@@ -43,7 +43,7 @@ jobs:
       - name: Build and Test
         run: sh ci/aarch64-linux-release.sh
   x86_64-macos-release:
-    runs-on: "macos-11"
+    runs-on: "macos-13"
```

```suggestion
    runs-on: "macos-12"
```

the CI makes sure things work on the oldest supported system